docs/identity/saas-apps/observepoint-web-assurance-tutorial.md

```diff
@@ -121,32 +121,32 @@ In this section, you'll create a test user in the Microsoft Entra admin center c
 
 ### Assign the Microsoft Entra ID test user
 
-In this section, you'll enable B.Simon to use Microsoft Entra single sign-on by granting access to ObservePoint Web Assurance.
+In this section, you'll enable B.Simon to use Microsoft Entra single sign-on by granting access to ObservePoint.
 
 1. Sign in to the [Microsoft Entra admin center](https://entra.microsoft.com) as at least a [Cloud Application Administrator](~/identity/role-based-access-control/permissions-reference.md#cloud-application-administrator).
-1. Browse to **Identity** > **Applications** > **Enterprise applications** > **ObservePoint Web Assurance**.
+1. Browse to **Identity** > **Applications** > **Enterprise applications** > **ObservePoint**.
 1. In the app's overview page, select **Users and groups**.
 1. Select **Add user/group**, then select **Users and groups** in the **Add Assignment** dialog.
    1. In the **Users and groups** dialog, select **B.Simon** from the Users list, then click the **Select** button at the bottom of the screen.
    1. If you are expecting a role to be assigned to the users, you can select it from the **Select a role** dropdown. If no role has been set up for this app, you see "Default Access" role selected.
    1. In the **Add Assignment** dialog, click the **Assign** button.
 
-## Configure ObservePoint Web Assurance SSO
+## Configure ObservePoint SSO
 
-To configure single sign-on on **ObservePoint Web Assurance** side, you need to send the downloaded **Federation Metadata XML** and appropriate copied URLs from Microsoft Entra admin center to [ObservePoint Web Assurance support team](mailto:support@observepoint.com). They set this setting to have the SAML SSO connection set properly on both sides.
+To configure single sign-on on **ObservePoint** side, you need to send the downloaded **Federation Metadata XML** and appropriate copied URLs from Microsoft Entra admin center to [ObservePoint  support team](mailto:support@observepoint.com). They set this setting to have the SAML SSO connection set properly on both sides.
 
-### Create ObservePoint Web Assurance test user
+### Create ObservePoint test user
 
-In this section, you create a user called B.Simon in ObservePoint Web Assurance. Work with [ObservePoint Web Assurance support team](mailto:support@observepoint.com) to add the users in the ObservePoint Web Assurance platform. Users must be created and activated before you use single sign-on.
+In this section, you create a user called B.Simon in ObservePoint . Work with [ObservePoint support team](mailto:support@observepoint.com) to add the users in the ObservePoint platform. Users must be created and activated before you use single sign-on.
```

Comment:
On this line, remove extra space after "ObservePoint" and before "."

Reply:
@djsmith42 thanks for catching that!